handler/src/main/java/io/netty/handler/ssl/OpenSslClientContext.java

```diff
@@ -247,24 +247,18 @@ public OpenSslClientContext(File trustCertCollectionFile, TrustManagerFactory tr
                     }
                     final X509TrustManager manager = chooseTrustManager(trustManagerFactory.getTrustManagers());
 
+                    // IMPORTANT: The callbacks set for verification must be static to prevent memory leak as
```

Comment:
it may also be nice for `SSLContext.setCertVerifyCallback`'s javadocs to declare this "limitation" as this is where the issue originates from.